### PR TITLE
Alias of ban is now yeet

### PR DESCRIPTION
**Pull request by austinmccalley**

### CODE REVIEW

**Comment by lostdesign** [comment]
Denied, should not be an alias but replace .ban :wew:

**Comment by AndresSweeneyRios** [comment]
Why not both?

**Comment by austinmccalley** [comment]
Fixing it

**Comment by FunctionDJ** [comment]
+1

**Comment by redxtech** [comment]
oh god

**Comment by austinmccalley** [comment]
Update -- Yeet is now its own command and ban no longer has the alias yeet

File: src/commands/yeet.js [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+
+      // Save the days arg. If it doesn't exist, default to 0. If it isn't an int, default to the amount specified in the config.
+      if ((args.length > (rmIndex + 1)) && !isNaN(parseInt(args[rmIndex + 1]))) {
+        days = parseInt(args[rmIndex + 1])
+        args.splice(rmIndex, 2)
+      } else {
+        days = banMsgDelete
+      }
+
+      // Save the args remaining after the first. If there isn't more than one arg, default to 'banned by devmod.'.
```
Comment (FunctionDJ) [inline src/commands/yeet.js:57]: 'banned by devmod' what the fuck is this garbage documentation

File: src/commands/yeet.js [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+      const reason = args.length > 1 ? args.slice(1).join(' ') : 'yeeted by devmod.'
+
+      try {
+        // Send the user a DM letting them know they've been banned.
```
Comment (FunctionDJ) [inline src/commands/yeet.js:61]: *yeeted

File: src/commands/yeet.js [hunk 3]
```diff
@@ -0,0 +1,134 @@
...
+        })
+      } catch (err) {
+        await logError('Yeet', 'Failed to send member message', err, message)
+      }
+
+      // Save some info about the staff member.
+      const staffMember = message.member
+
+      try {
+        // Log the ban to the bans channel.
```
Comment (FunctionDJ) [inline src/commands/yeet.js:89]: *yeets

File: src/commands/yeet.js [hunk 4]
```diff
@@ -0,0 +1,134 @@
...
+              },
+              timestamp: new Date()
+            }
+          })
+      } catch (err) {
+        await logError('Yeet', 'Failed to log yeet to channel', err, message)
+      }
+
+      try {
+        // Ban the member.
```
Comment (FunctionDJ) [inline src/commands/yeet.js:116]: *Yeet

**Comment by redxtech** [comment]
I'm going to close this PR, but I will re-work the ban command to use `yeet` instead of `ban` throughout if it's called by that alias.

That way avoids significant code duplication.

**Comment by redxtech** [comment]
we back

**Pull request merged**